src/Jellyfin.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -120,8 +120,18 @@ public ImageDimensions GetImageSize(string path)
         if (extension.Equals(".svg", StringComparison.OrdinalIgnoreCase))
         {
             var svg = new SKSvg();
-            svg.Load(path);
-            return new ImageDimensions(Convert.ToInt32(svg.Picture.CullRect.Width), Convert.ToInt32(svg.Picture.CullRect.Height));
+            try
+            {
+                svg.Load(path);
+                return new ImageDimensions(Convert.ToInt32(svg.Picture.CullRect.Width), Convert.ToInt32(svg.Picture.CullRect.Height));
+            }
+            catch (FormatException skiaColorException)
+            {
+                // this exception is known to be thrown on vector images that define custom styles
+                // skia svg is not able to parse that and as the repository is quite stale and has not received updates we just catch them
+                _logger.LogDebug(skiaColorException, "There was a issue loading the requested svg file");
+                return new ImageDimensions(0, 0);
```

Comment:
What side-effect happens with a 0,0 image dimensions?

Reply:
I am not 100% sure but it is the default used below for non-successfull return codes:
https://github.com/jellyfin/jellyfin/blob/c2684e537c5e52b2b4164565bee107d5aaeb8380/src/Jellyfin.Drawing.Skia/SkiaEncoder.cs#L145

Reply:
Edit:
Seems like 0,0 is handled as a special case in 5/6 callers as an default value specifically.

Reply:
The 6th case also defaults to 0,0 on exceptions so i assume its fine

Reply:
```suggestion
                return default;
```

This should work too and is maybe a bit more clear

Reply:
@Bond-009 should i adjust related instances aswell? I mean if that style then everywhere.

Reply:
That would be nice